build.gradle

```diff
@@ -68,11 +70,11 @@ sourceSets {
         output.dir(generatedVersionDir, builtBy: 'generateVersionProperties')
     }
 }
-spotless {
+/*spotless {
     java {
         googleJavaFormat()
     }
-}
+}*/
```

Comment:
Just curious what was the reason to comment out the `spotless` section 🤔

Reply:
It fails to run. There seem to be some version incompatibility issues / bugs. I'll do the necessary fixes to re-enable it on another PR as I didn't want to keep this waiting. It's passing the tests now, which weren't working correctly before.